classes/mender-boot.bbclass

```diff
@@ -0,0 +1,5 @@
+EXTRA_IMAGEDEPENDS += "u-boot u-boot-fw-utils"
+PREFERRED_VERSION_u-boot = "v2015.10%"
+PREFERRED_VERSION_u-boot-fw-utils = "v2015.10%"
+
+IMAGE_INSTALL_append = " u-boot-fw-utils"
```

Comment:
It is `mender-full` you need to include to get everything. But maybe it would be better if this class was called `mender-uboot`, not `mender-boot`.